doc/classes/String.xml

```diff
@@ -143,15 +143,15 @@
 				GD.Print("team".Contains("I"));  // Prints False
 				[/csharp]
 				[/codeblocks]
-				If you need to know where [param what] is within the string, use [method find]. See also [method containsn].
+				If you need to know where [param what] is within the string, use [method find]. See also [method containsn] and [method is_subsequence_of].
```

Comment:
Not 100% sure if I'd link to `is_subsequence_of`, since its functionality is rather niché, and probably not relevant to people looking at `contains`.